framework/src/main/java/org/tron/core/Wallet.java

```diff
@@ -844,13 +851,7 @@ public GrpcAPI.GetAvailableUnfreezeCountResponseMessage getAvailableUnfreezeCoun
     }
     long now = dynamicStore.getLatestBlockHeaderTimestamp();
 
-    List<UnFreezeV2> unfrozenV2List = accountCapsule.getInstance().getUnfrozenV2List();
-    long getUsedUnfreezeCount = unfrozenV2List
-            .stream()
-            .filter(unfrozenV2 ->
-                    (unfrozenV2.getUnfreezeAmount() > 0
```

Comment:
`unfrozenV2.getUnfreezeAmount() > 0` no longer needed?

Reply:
OK，I'll remove it.